src/template.rs

```diff
@@ -1,10 +1,44 @@
 use Renderable;
 use context::Context;
-use filters::{size, upcase, downcase, capitalize, minus, plus, times, divided_by, ceil, floor,
-              round, prepend, append, first, last, pluralize, replace_first, replace, date, sort,
-              slice, modulo, escape, escape_once, remove_first, remove, strip_html, truncatewords};
```

Comment:
That was just because it makes it match the list of filters in the `impl Renderable ...` section, and it made it easier to see where/what was added.  I don't have strong feelings about it either way, so I'll put it back to a grouped version (ordered alphabetically).